ETC.tsv

```diff
@@ -86,7 +86,7 @@ xETC_20150317_000085	Its stem head can be used for various purposes, such as rel
 ETC_20150317_000086	Big Gribaru
 ETC_20150317_000087	Thorn Forest Griba
 ETC_20150317_000088	Matsum
-ETC_20150317_000089	Whether the Meshmu originated from a plant, animal or mineral is not important. What you should know is that it is dangerous.
+xETC_20150317_000089	Whether the Meshmu originated from a plant, animal or mineral is unimportant. What one should know is that they are dangerous.
```

Comment:
One -> you
"they are" would be implying multiply, since we don't know the context of this sentence, leave it at it is. It is refers to a specific person/place.